campers.html

```diff
@@ -18,29 +18,22 @@
       <div class="include" data-include="header">
       </div>
       <div id="content" >
-        <div class="camper_blogs">
+        <div class="campers_page">
           <div class="subtitle">
-            <h2>Blogs and Projects</h2>
+            <div class="blog_toggle" id="blog_toggle"><h2>Blogs</h2></div><div class="projects_toggle" id="projects_toggle"><h2>Projects</h2></div>
```

Comment:
When they are on separate lines in your source file, the second line is preceded by several spaces:

```html
    <div class="blog_toggle" id="blog_toggle"><h2>Blogs</h2></div>
    <div class="projects_toggle" id="projects_toggle"><h2>Projects</h2></div>
```

Multiple spaces in your document are converted to a single space in the browser.

```
<p>This       .</p>
```
becomes
```
This .
```
on the screen.

To prevent these extra spaces from disrupting your visuals, you need to set the `font-size` of the containing element to `0px`. Here is an example:

* https://jsfiddle.net/ndxu4qa7